src/main/java/org/broadinstitute/hellbender/tools/exome/PlotACNVResults.java

```diff
@@ -22,6 +22,14 @@
     protected String snpCountsFile;
 
     @Argument(
+            doc = "File of tangent normalized coverage of targets",
```

Comment:
tangent-normalized (being a stickler for hyphenating compound modifiers is something i picked up from my adviser...apologies)